test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -45,6 +45,16 @@ struct UcxAmDesc {
     const ucp_am_recv_param_t    *_param;
 };
 
+void *ucx_am_get_data(const UcxAmDesc &desc)
+{
+    return desc._data;
+}
+
+bool ucx_am_is_rndv(const UcxAmDesc &desc)
+{
+    return desc._param->recv_attr & UCP_AM_RECV_ATTR_FLAG_RNDV;
```

Comment:
pls make this function static inline protected, inside ucx_context class

Reply:
Making it inline needs moving UcxAmDesc definition to ucx_wrapper.h. @brminich What is your take?

Reply:
ok

Reply:
done